src/test/objectstore/test_memstore_clone.cc

```diff
@@ -178,7 +178,7 @@ int main(int argc, char** argv)
   // default to memstore
   vector<const char*> defaults{
     "--osd_objectstore", "memstore",
-    "--osd_data", "memstore.test_temp_dir",
```

Comment:
The new standard is directory names are like "testname.test_temp_dir".  So this could be "memstore_clone.test_temp_dir" or something shorter.